MusicTaste/MusicTaste/MatchingAlgorithm.m

```diff
@@ -16,38 +16,34 @@ + (instancetype)shared {
     return sharedManager;
 }
 
-+ (void) lookForMatches{
+- (void) lookForMatches{
     NSMutableDictionary *matchDict = [NSMutableDictionary new];
     PFObject *music = [PFObject objectWithClassName:@"Music"];
     PFQuery *query = [PFQuery queryWithClassName:@"Music"];
     PFUser *current = [PFUser currentUser];
     //Goes through Query
     [query findObjectsInBackgroundWithBlock:^(NSArray *spotifyData, NSError *error) {
         for (PFObject *user in spotifyData){
-            //We make sure we are not comparing the same object
-            if (![user.objectId isEqual:music.objectId]){
-                //Calls the compare function to compare each user
-                [MatchingAlgorithm compare:user withDictionary:matchDict withData:user];
-            }
+            //Calls the compare function to compare each user
+            [self compareSpotifyData:user withData:spotifyData];
         }
+        //sets the matches as an integer in the user's database
+        [current saveInBackgroundWithBlock:^(BOOL succeeded, NSError * _Nullable error) {}];
     }];
-    //sets the matches as an integer in the user's database
-    NSArray *matches = [[[matchDict keysSortedByValueUsingSelector:@selector(compare:)] reverseObjectEnumerator] allObjects];
```

Comment:
Unnecessary to set the matches here when I want to set the matches for each user